docs/development/i18n.rst

```diff
@@ -359,7 +359,7 @@ Inspect the screenshots in the directory ``securedrop/tests/pageslayout/screensh
 and make sure that their content corresponds to the expected version of the
 codebase.
 
-`Obtain your API key <https://weblate.securedrop.org/accounts/profile/#api>`__
+`Obtain your API key <https://weblate.securedrop.org/accounts/login/?next=/accounts/profile/#api>`__
```

Comment:
Did you intentionally modify this URL?

Reply:
yeah. It was coming up as a fail in the docs link check

Reply:
Got it, thanks for clarifying. :) The issue here is that this link requires being logged into Weblate. I'd prefer not to rely on Weblate's login redirect logic just to make the link checker happy. You can just add it to the ignore-list for the linkcheck, here, which is generally what we do for links that require authentication: 

https://github.com/freedomofpress/securedrop-docs/blob/main/docs/conf.py#L320-L326

Reply:
ok. reverted back